resources/home/dnanexus/run_workflows/run_workflows.py

```
@@ -398,7 +398,7 @@ def main():
                             "This run was processed automatically by "
                             "eggd_conductor: "
                         ),
-                        url=f"http://{os.environ.get('conductor_job_url')}",
+                        url=f"https://{os.environ.get('conductor_job_url')}",
```

Ruff (0.12.2)

Use capitalized environment variable `CONDUCTOR_JOB_URL` instead of `conductor_job_url`

(SIM112)